src/crawler/crawlers/arso_weather_slovenija/crawler.js

```diff
@@ -0,0 +1,273 @@
+                        for (const timestamps of Object.keys(weatherData[station].points[uuids])) {
+                            for (const sensorKey of Object.keys(weatherData[station].points[uuids][timestamps])) {
+                                for (const existingSensors of (Object.values(sensors))) {
+                                    if (existingSensors.uuid == uuids.slice(1) + ' - ' + sensorKey.slice(1)) {
+                                        const SQL = this.SQLUtils.insertMeasurementSQL(existingSensors.id, (parseInt(timestamps.slice(1))-89411160)*60, weatherData[station].points[uuids][timestamps][sensorKey]);
+                                        try {
+                                            this.SQLUtils.processSQL(SQL);
+                                        } catch (e) {
+                                            console.log('Error in:', SQL);
+                                        };
+                                    }
+                                }
+                            }
+                        }
+                    }
+
+                    // Sets state if already exists
+                    let fromTime = this.config.start_time;
+                    if (this.state != {}) {
+                        if (this.state[name] !== undefined) {
+                            fromTime = moment.unix(this.state[name].lastRecord).toISOString(true);
+                        }
+                    }
+
+                    // Update datalake repository with the crawled data
+                    if (fromTime !== lastMeasure) {
+                        CrawlerUtils.saveToDataLake(stationData, fromTime, {
+                            dir: this.config.id,
+                            type: this.config.log_type,
+                            name: name,
+                        });
+                    }
+
+
+                    // Update the state with the last crawled timestamp
+
+                    this.updateState(stationList, lastUnix);
+
+
+                    // Write final state
+                    CrawlerUtils.saveState(__dirname, this.state);
+                }
+                // Crawling next date
+                this.timeInterval[0] = moment(this.timeInterval[0]).add(1, 'day').format('YYYY-MM-DD');
+            }
+        } catch (e) {
+            console.log('ERROR:', e);
+        }
+
+        console.log('Finishing crawl: ' + this.config.id);
+    }
+
+
+    async getLocationData(base_loc = this.config.url_locations,
+        timeInterval = this.timeInterval,
+        types = this.config.station_type) {
+        const url = base_loc + 'd1=' + timeInterval[0] + '&d2=' + timeInterval[1] + '&type=' + types;
+        const body = await CrawlerUtils.getURL(url);
+        const $ = cheerio.load(body, {xmlMode: true});
+        const stringData = $('pujs').text().slice(17, -2);
+        const locationData = JSON.parse(stringData.replace(/(\w+:)|(\w+ :)/g, function(s) {
```

Comment:
Comment, what this does, it is not intuitive to read regexp; instead of function(s) { try using () => {